hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleAccountState.java

```diff
@@ -69,7 +69,8 @@
 	static final int RELEASE_0180_VERSION = 7;
 	static final int RELEASE_0210_VERSION = 8;
 	static final int RELEASE_0220_VERSION = 9;
-	static final int RELEASE_0250_VERSION = 10;
+	static final int RELEASE_0230_VERSION = 10;
+	static final int RELEASE_0250_VERSION = 11;
```

Comment:
If it's okay to have 250 and 240 as same version number, please call this:
```
RELEASE_0240_OR_0250_VERSION = 11;
```

Reply:
And also add a comment explaining why it's okay to combine the two.

Reply:
As noted above, our current naming convention only increments a `SelfSerializable`'s version number in a release where it changes.